src/Expression.php

```diff
@@ -111,6 +109,27 @@ public function offsetGet($offset) {
     }
 
     /**
+     * Use this instead of "new Expression()" if you want to automatically bind
+     * expression to the same connection as the parent.
+     */
+    public function expr($expr, $options = [])
+    {
+        $options['connection'] = $this->connection;
+        $class = get_class($this);
```

Comment:
I was thinking about duplicating same class, but not sure how to approach this properly